### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via  
```
codespell -q 3 -L aci,ake,aline,alle,alledges,alocation,als,ang,anid,ba,beginn,behaviour,bloaded,byteorder,calculater,cancelled,cancelling,cas,cascade,centimetre,childs,colour,colours,commen,connexion,currenty,dof,doubleclick,dum,eiter,elemente,ende,feld,finde,findf,freez,hist,iff,indicies,initialisation,initialise,initialised,initialises,initialisiert,ist,kilometre,lod,mantatory,methode,metres,millimetre,modell,nd,noe,normale,normaly,nto,numer,oder,orgin,orginx,orginy,ot,pard,pres,programm,que,recurrance,rougly,seperator,serie,sinc,strack,substraction,te,thist,thru,tread,uint,unter,vertexes,wallthickness,whitespaces -S ./.git,*.po,*.ts,./ChangeLog.txt,./src/3rdParty,./src/Mod/Assembly/App/opendcm,./src/CXX,./src/zipios++,./src/Base/swig*,./src/Mod/Robot/App/kdl_cp,./src/Mod/Import/App/SCL,./src/WindowsInstaller,./src/Doc/FreeCAD.uml
```

### CODE REVIEW

**Comment by vocx-fc** [comment]
Note to self: this will probably cause conflicts with #3082. Need to rebase if it is merged first.

File: src/Mod/Fem/femsolver/calculix/writer.py [hunk 1]
```diff
@@ -1160,7 +1160,7 @@ def write_constraints_pressure(self, f):
             f.write("*DLOAD\n")
             for ref_shape in femobj["PressureFaces"]:
                 # the loop is needed for compatibility reason
-                # in depretiated method get_pressure_obj_faces_depreciated
+                # in depreciated method get_pressure_obj_faces_depreciated
```
Comment (luzpaz) [inline src/Mod/Fem/femsolver/calculix/writer.py]: shouldn't this be 'deprecated' ?  
` in **deprecated** method get_pressure_obj_faces_**deprecated**`

File: src/Mod/Fem/femtools/femutils.py [hunk 2]
```diff
@@ -120,7 +120,7 @@ def is_derived_from(obj, t):
 def get_pref_working_dir(solver_obj):
     """ Return working directory for solver honoring user settings.
 
-    :throws femtools.erros.MustSaveError:
+    :throws femtools.errors.MustSaveError:
```
Comment (luzpaz) [inline src/Mod/Fem/femtools/femutils.py]: @berndhahnebach this should fix something :stuck_out_tongue_winking_eye:

File: src/Mod/Fem/femsolver/calculix/writer.py (hunk 1 above)
Comment (luzpaz) [reply src/Mod/Fem/femsolver/calculix/writer.py]: Looks like there is quite a few mentions in 
```
src/Mod/Fem/femmesh/meshtools.py:# ***** depreciated method for retrieving pressure faces *****************************************
src/Mod/Fem/femmesh/meshtools.py:def get_pressure_obj_faces_depreciated(
src/Mod/Fem/femmesh/meshtools.py:    # see comment on get_pressure_obj_faces_depreciated in the regard of getccxVolumesByFace()
src/Mod/Fem/femsolver/writerbase.py:        # depreciated version
src/Mod/Fem/femsolver/writerbase.py:            femobj["PressureFaces"] = meshtools.get_pressure_obj_faces_depreciated(
src/Mod/Fem/femsolver/writerbase.py:            # the data model is for compatibility reason with depreciated version
src/Mod/Fem/femsolver/writerbase.py:            # get_pressure_obj_faces_depreciated returns the face ids in a tuple per ref_shape
src/Mod/Fem/femsolver/writerbase.py:            # some_string was the reference_shape_element_string in depreciated method
src/Mod/Fem/femsolver/calculix/writer.py:                # in deprecated method get_pressure_obj_faces_depreciated
```

File: src/Mod/Fem/femtools/femutils.py (hunk 2 above)
Comment (luzpaz) [reply src/Mod/Fem/femtools/femutils.py]: thanks for thoroughly investigating that and also _killing my buzz_ :stuck_out_tongue:

File: src/Mod/Fem/femsolver/calculix/writer.py (hunk 1 above)
Comment (berndhahnebach) [reply src/Mod/Fem/femsolver/calculix/writer.py]: Why does this command does not find thjem?

`codespell -q 2 -S *.ts -L childs,dof,dum,methode,nd,normaly,uint,vertexes,freez src/Mod/Fem/`

Comment (luzpaz) [reply src/Mod/Fem/femsolver/calculix/writer.py]: Two reasons:  
1.  I used a different dictionary file from https://github.com/orbitcowboy/codespell_dictionary (though it has many errors in it, it's still useful)  
2. codespell still isn't sophisticated enough to encompass usecases and edgecases that it encounters.

**Comment by berndhahnebach** [comment]
merged

**Comment by berndhahnebach** [comment]
> 
> 
> Note to self: this will probably cause conflicts with #3082. Need to rebase if it is merged first.

@vocx-fc will you go for the rebase ?

**Comment by vocx-fc** [comment]
Yes, of course.

File: src/Mod/Draft/DraftGeomUtils.py [hunk 3]
```diff
@@ -1346,7 +1346,7 @@ def offsetWire(wire,dvec,bind=False,occ=False,widthList=None, offsetMode=None, a
         if i == 0:
             if alignListC[0] == 'Center':
                 delta = DraftVecUtils.scaleTo(delta, delta.Length/2)
-            #No need to do anything for 'Left' and 'Rigtht' as orginal dvec have set both the direction and amount of offset correct
+            #No need to do anything for 'Left' and 'Rigtht' as original dvec have set both the direction and amount of offset correct
```
Comment (vocx-fc) [inline src/Mod/Draft/DraftGeomUtils.py]: @luzpaz LOL. I just noticed that you missed the word `Rigtht`.

Comment (luzpaz) [reply src/Mod/Draft/DraftGeomUtils.py]: 'Typo saturation' :lol:

Comment (luzpaz) [reply src/Mod/Draft/DraftGeomUtils.py]: I can't atm but can someone make a PR to the codespell dictionary?

**Pull request merged**